src/index.html

```diff
@@ -24,9 +24,12 @@ <h2 class="sub-header">Recent scores</h2>
       </div>
       <form class="form-score">
         <h2 class="sub-header">Add your score</h2>
-        <input type="text" placeholder="Your name" class="name-input" /><br><br>
-        <input type="text" placeholder="Score" class="score-input" /><br><br>
-        <button id="submit-score" type="submit">Submit</button>
+        <input type="text" placeholder="Your name" class="name-input input-field" /><br><br>
+        <input type="text" placeholder="Score" class="score-input input-field" /><br><br>
+        <div>
+
+          <button id="submit-score" type="submit">Submit</button>
```

Comment:
- [ ] Kindly align the submit button in the form with the input fields, kindly note that in this milestone we have to follow the given wireframe design.

![leaderboard](https://github.com/majdalkilany/Leaderboard-/assets/53124548/7315d824-9b36-456b-8395-b04ad9d72899)

![submit](https://github.com/majdalkilany/Leaderboard-/assets/53124548/2c498076-1c31-4871-a3a4-0c6c0894b660)